data/OpenRA_Red_Alert

```diff
@@ -0,0 +1,2 @@
+https://github.com/fusion809/OpenRA/releases/download/untagged-5c8c5c74943bcc8b5588/OpenRA-Red-Alert-devel-x86_64.AppImage
```

Comment:
Please point this to https://github.com/OpenRA/OpenRA/releases/download/release-20200202/OpenRA-Red-Alert-x86_64.AppImage or similar.

Reply:
This line is the _only_ input here. All other changes you requested need to be made inside that AppImage, as we are merely extracting the information from there.